src/components/dashboard/Receive.js

```diff
@@ -20,19 +20,19 @@ export type ReceiveProps = {
 
 const RECEIVE_TITLE = 'Receive G$'
 const SHARE_TEXT = 'Share your wallet link'
-let shareLink
 const Receive = ({ screenProps, styles, ...props }: ReceiveProps) => {
   const profile = GDStore.useStore().get('profile')
   const { account, networkId } = goodWallet
-
+  const [shareLink, setShareLink] = useState(null)
   const [showErrorDialog] = useErrorDialog()
   const amount = 0
   const reason = ''
   const codeObj = useMemo(() => generateCode(account, networkId, amount, reason), [account, networkId, amount, reason])
   const share = useMemo(() => generateReceiveShareObject(codeObj, amount, '', profile.fullName), [codeObj])
   useEffect(() => {
```

Comment:
I dont see why we need usememo+useeffect all the parameters in this screen seem to be fixed, not dynamic, and not async. we can leave usememo but useffect  doesnt seem to be required

Reply:
done in https://github.com/GoodDollar/GoodDAPP/pull/1030/commits/9a49a701fbbedf30ae06f364ad1ba791527133a2